docs/tables/{custom_resource_name.group_name}.md

```diff
@@ -0,0 +1,556 @@
+                renewalTime:
+                  description: RenewalTime is the time at which the certificate will be next renewed. If not set, no upcoming renewal is scheduled.
+                  type: string
+                  format: date-time
+                revision:
+                  description: "The current 'revision' of the certificate as issued. \n When a CertificateRequest resource is created, it will have the `cert-manager.io/certificate-revision` set to one greater than the current value of this field. \n Upon issuance, this field will be set to the value of the annotation on the CertificateRequest resource used to issue the certificate. \n Persisting the value on the CertificateRequest resource allows the certificates controller to know whether a request is part of an old issuance or if it is part of the ongoing revision's issuance by checking if the revision value in the annotation is greater than this field."
+                  type: integer
+      served: true
+      storage: true
+```
+
+```bash
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: temporal-w-spcloudt6t6sk7toegg-tls
+  labels:
+    workspace-id: w_spcloudct6t6sk7toegg
+    identity-id: o_spcloud6un5ogrh1bc0
+    region: apse1
+    shard: "0001"
+    workspace-pluginset: "20220317010004"
+    app.kubernetes.io/component: steampipe-workspace-db
+    app.kubernetes.io/managed-by: steampipe-api
+    app.kubernetes.io/part-of: steampipe-cloud
+    app.kubernetes.io/instance: w-spcloudct6t6sk7toegg
+    app.kubernetes.io/version: 0.13.3-workspace-spcloud.20220317010004
+spec:
+  secretName: temporal-w-spcloudt6t6sk7toegg-tls
+  duration: 87600h # 10 years
+  renewBefore: 360h # 15d
+  usages:
+    - server auth
+    - client auth
+  issuerRef:
+    name: ca-issuer-temporal
+    kind: ClusterIssuer
+  dnsNames:
+    - w-spcloudt6t6sk7toegg
+```
+
+### Inspect the table structure
+
+List all tables:
+
+```sql
+.inspect kubernetes;
++---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| table                                 | description                                                                                                                                                      |
++---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| certificates.cert-manager.io          | Represents Custom resource certificates.cert-manager.io.                                                                                                         |
+| crontabs.stable.example.com           | Represents Custom resource crontabs.stable.example.com.                                                                                                          |
```

Comment:
Are these tables created from the sample definitions above? We should only show tables in this example assuming a user uses the CRDs above.

Reply:
certificates.cert-manager.io created from the sample above; I have removed the crontab one.